pkg/asset/agent/image/ignition.go

```diff
@@ -248,6 +248,7 @@ func (a *Ignition) Generate(dependencies asset.Parents) error {
 func getDefaultEnabledServices() []string {
 	return []string{
 		"agent-interactive-console.service",
+		"agent-interactive-console-serial@.service",
```

Comment:
q: how does it get activated (ie who's passing the instance parameter)?

Reply:
Because it has `WantedBy=serial-getty@.service` it gets started with the same parameter as serial-getty when the latter is started.

The go API here is a little weird, but when you look at the Ignition it generates, the effect is that we're just setting enabled=true in the systemd unit entry with this filename.

Reply:
Ok thanks!